creator-node/src/config.js

```diff
@@ -488,7 +488,7 @@ const config = convict({
     doc: 'Max concurrency of SyncQueue',
     format: 'nat',
     env: 'syncQueueMaxConcurrency',
-    default: 50
+    default: 30
```

Comment:
changed to 30 because figured 30 of each recurring/manual is better than 50 total